src/pybind/mgr/orchestrator.py

```diff
@@ -34,7 +34,15 @@
 logger = logging.getLogger(__name__)
 
 
-HostSpec = namedtuple('HostSpec', ['hostname', 'network', 'name'])
+class HostSpec(namedtuple('HostSpec', ['hostname', 'network', 'name'])):
```

Comment:
could you move this change to another commit?

Reply:
Eh, but this change is also required by mypy!?

Reply:
ahh, i didn't realize this. because mypy needs a type with the `__str__` magic method?

Reply:
done